src/test/java/com/serenitydojo/WhenWritingReadableAssertions.java

```diff
@@ -95,38 +96,40 @@ public void whenAListContainsAValue() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains "red"
+        assertThat(colors).contains("red");
     }
 
     @Test
     public void whenAListContainsASetOfValues() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains "red" and "blue
-        assertThat(true).isFalse();
+        assertThat(colors).contains("red","blue");
     }
 
     @Test
     public void whenAListHasASpecificSize() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains 3 elements
-        assertThat(true).isFalse();
+        assertThat(colors).hasSize(3);
     }
 
     @Test
     public void whenAListContainsExactlyASetOfValues() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains "red","green" and "blue"
-        assertThat(true).isFalse();
+        assertThat(colors).containsExactly("red", "green", "blue");
+
     }
 
     @Test
     public void whenAListContainsExactlyASetOfValuesInAnyOrder() {
         List<String> colors = Arrays.asList("red","green","blue");
 
         // Check that the list contains "red","blue" and "green" in any order
-        assertThat(true).isFalse();
+        assertThat(colors).containsExactlyInAnyOrder("red", "green", "blue");
```

Comment:
```suggestion
        assertThat(colors).containsExactlyInAnyOrder("red", "blue", "green");
```
the point is to showcase that the order can be mangled but it still asserts successfully

Reply:
Wow, so true. I missed this. I'll correct it.